### DISCUSSION TITLE
Yoda 1.9 Meta data schema overview and comparison

### DISCUSSION THREAD

**Discussion by pradeep10kumar** [UtrechtUniversity/yoda#334] [Q&A] [answered]
Hello, where can I locate information on the details, features, field names, pros and cons, and guidance on when to use each metadata schema available in version 1.9? Additionally, I'm curious about which schema in version 1.9 is comparable to those in version 1.8.

**Comment by pradeep10kumar** [comment]
Can I have some details?

**Reply by stsnel** [reply, marked as answer]
Hello Pradeep,

Yoda includes a default metadata schema that aims to be generic, so a logical place to start would be to see if this schema works for you. You can find additional information about the default metadata schema on the UU Yoda website: https://www.uu.nl/en/research/yoda/guide-to-yoda/i-am-using-yoda/documenting-your-data . 

The other metadata schemas that are included in Yoda are generally custom schemas for particular use cases in Utrecht University, and therefore not designed for use in other organizations.

If the default metadata schema does not meet your requirements, you can consider developing one or more custom metadata schemas, based on the default or core metadata schema. Custom schemas would have to be designed, supported and maintained in your own organization. 

A high-level overview of the most important changes in Yoda 1.9, including metadata schema changes, can be found in the release notes of Yoda 1.9 (https://github.com/UtrechtUniversity/yoda/blob/development/docs/release-notes/release-1.9.md). If you want to examine the changes in detail, you could diff the schema files of two schema versions in the `schemas` directory of the yoda-ruleset repository (https://github.com/UtrechtUniversity/yoda-ruleset). For example: `diff schemas/default-2/metadata.json schemas/default-3/metadata.json | less`.

Hope this helps.

Best regards,
Sietse

**Comment by pradeep10kumar** [comment]
Hello Sietse,

I appreciate your response! The information you provided is quite helpful. To clarify, could you please explain the differences between the default, default0, default1, default2, etc., metadata forms?

Best regards,
Pradeep

**Reply by stsnel** [reply]
Hi Pradeep,

In metadata schema identifiers, the number suffix refers to the schema version. So `default-2` is version 2 of the `default` metadata schema, for example. You should use the latest version of the metadata schema that is available for your Yoda version. For Yoda 1.9.x, that is `default-3`. You can find more information about the identifier format here: https://github.com/UtrechtUniversity/yoda/blob/development/docs/design/metadata/adr-schema-identifiers.md .

The `default` collection in the metadata schema collection is a special case: this refers to the default metadata schema version that has been configured on the environment. More information about the way metadata schema configuration works can be found here: https://github.com/UtrechtUniversity/yoda/blob/development/docs/design/metadata/schema-configuration.md

Best,
Sietse
